benefits/eligibility/templates/eligibility/index.html

```diff
@@ -0,0 +1,29 @@
+{% extends "core/base.html" %}
+
+{% block page_title %}
+  {{ page.title }} |
+{% endblock page_title %}
+
+{% block main_content %}
+  <div class="container">
+    <div class="row justify-content-center">
+      <div class="col-lg-10">
+        {% block icon_title %}
```

Comment:
Also this `block` - it doesn't exist in the `base.html` template and there are no child templates inheriting from this one...?